src/Layout/redist/dnx.ps1

```diff
@@ -0,0 +1,6 @@
+# dnx.ps1
```

Comment:
Should we add a `#Requires -Version 6.0`
to make sure powershell supports `Split-Path`?

@Args was also added in 3.0 (2012 powershell).

Reply:
It looks like `Split-Path` has been around since PowerShell 1.0.  We don't want to require version 6 of PowerShell, I think that's the .NET Core version that you have to install separately.  I think we want this script to work on the PowerShell that's included with Windows.  As far as I can tell it does.  The lowest [version of Windows we support](https://github.com/dotnet/core/blob/main/release-notes/8.0/supported-os.md) is Server 2012 which has PowerShell 3, so I think we're good.